d2l-dom.html

```diff
@@ -0,0 +1,62 @@
+<script>
+(function() {
+	'use strict';
+
+	var Dom = {
+
+		getComposedChildren: function(node) {
+
+			if (!node) {
+				return null;
+			}
+			if (node.nodeType !== 1 && node.nodeType !== 9 && node.nodeType !== 11) {
```

Comment:
Only try to get children for elements (1), documents (9), and document fragments (11).  In walking shadow-DOM, the shadow-root is a fragment, hence why that nodeType is included here.